web/client/components/mapcontrols/search/__tests__/SearchBar-test.jsx

```diff
@@ -100,15 +109,26 @@ describe("test the SearchBar", () => {
     });
 
     it('test that options are passed to search action', () => {
-        let searchOptions = {displaycrs: "EPSG:3857"};
+        let searchOptions = {displaycrs: "EPSG:3857",
+            services: [
```

Comment:
can you add more than one service?

with at least one with no maxFeatures defined